jp/stripStopWords.js

```diff
@@ -0,0 +1,26 @@
+module.exports={
+	strip_stopwords: function(languageCode, inputText){
+		var fileSystem = require('fs');
+		var validLanguageCodes = fileSystem.readFileSync('../data/list.txt').toString().split('\n');
+		var languageCodeValid = validLanguageCodes.indexOf(languageCode);
+
+		if (languageCodeValid === -1){
+			return "Error: Not a valid language code"
+		}
+
+		var stopWords = fileSystem.readFileSync('../data/' + languageCode + '.txt').toString().split('\n');
```

Comment:
It could be an undefined exception if the specified language's stop words file does not exist. For example, there's 'kr' language in data/list.txt  but the data/kr.txt does not exist.

Reply:
That possibility didn't even cross my mind until you mentioned it! Great catch! I will add code to handle that case